src/components/MegadraftEditor.js

```diff
@@ -152,6 +154,23 @@ export default class MegadraftEditor extends Component {
     }
   }
 
+  getSidebarComponent = () => {
+    const { sidebarRendererFn, editorState } = this.props;
+    let SidebarComponent = null;
+    if(typeof sidebarRendererFn === "function") {
+      SidebarComponent = sidebarRendererFn(this.plugins, editorState);
```

Comment:
onChange will be passed from the upper level component. I have made an example in docs. 

Good catch about `return` statement. I'll do it.